reexec/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/moby/sys/reexec
+
+go 1.18
```

Comment:
Probably will work with any version of Go, but go1.18 is the oldest version we test against in this repository